test/jdk/java/foreign/TestBufferStackStress2.java

```diff
@@ -23,6 +23,10 @@
 
 /*
  * @test
+ *
+ * If the VM does not have continuations, then no VTs can ever be scheduled
+ * @requires vm.continuations
```

Comment:
The comment isn't quite right. Virtual threads will still be scheduled, it's just that the OS scheduler will be used as each virtual thread is backed by an OS thread.